src/module/actor/inventory/bulk.ts

```diff
@@ -80,7 +80,12 @@ export class InventoryBulk {
         return this.value.normal;
     }
 
-    static computeTotalBulk(items: PhysicalItemPF2e[], actorSize: Size): Bulk {
+    static computeTotalBulk(items: PhysicalItemPF2e[], actorOrSize: ActorPF2e | Size | null): Bulk {
+        const [actor, actorSize]: [ActorPF2e | null, Size] = !actorOrSize
+            ? [null, "med"]
+            : typeof actorOrSize === "string"
+              ? [null, actorOrSize]
+              : [actorOrSize, actorOrSize.size];
         items = this.#flattenNonStowing(items);
```

Comment:
imo just pass the actor and an an array of items

Reply:
Done